File: SAMPLES/CSHARP/test09_array.cs
using System;

namespace Test09
{
    class Program
    {
        static void Main()
        {
            int[] arr = new int[5];
            int   i;
            int   sum;

            arr[0] = 10;
            arr[1] = 20;
            arr[2] = 30;
            arr[3] = 40;
            arr[4] = 50;

            sum = 0;
            for (i = 0; i < 5; i++)
            {
                sum += arr[i];
            }
            Console.WriteLine(sum);

            for (i = 0; i < 5; i++)
            {
                Console.WriteLine(arr[i]);
            }
        }
    }
}

File: SAMPLES/JAVA/test09_array.java
public class test09_array {
    public static void main(String[] args) {
        int[] tab = new int[5];
        int i;
        int sum;

        tab[0] = 10;
        tab[1] = 20;
        tab[2] = 30;
        tab[3] = 40;
        tab[4] = 50;

        sum = 0;
        for (i = 0; i < 5; i++) {
            sum += tab[i];
        }
        System.out.println(sum);

        for (int x : tab) {
            System.out.println(x);
        }

        tab[2] = 99;
        System.out.println(tab[2]);
    }
}
